### DISCUSSION TITLE
Binding source of Image is not work

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#12928] [Q&A]
I tried to bind a WriteableBitmap property to Image source, when I change the WriteableBitmap property , the UI is not update.
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/4eee3bb4-47e9-474a-8f0d-432561b87b28)
 If I directly set to Image source, it can be changed, but this way can not crop a image.
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/38bf81ef-e9aa-4d1a-bdfc-1cd181042a6a)
  So how can I bind cropped bitmap to image source, and update it dynamically?

**Comment by timunie** [comment]
you need to invalidate the Visual after the Bitmap has changed. Until today, RTB doesn't support it but this feature is planned irrc.

**Reply by jeff9571** [reply]
what is RTB by the way?

**Reply by timunie** [reply]
RichTextBlock but now it was renamed to SelectableTextBlock iirc.
